tools/io_tracer_parser_tool.cc

```diff
@@ -0,0 +1,144 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//    This source code is licensed under both the GPLv2 (found in the
+//    COPYING file in the root directory) and Apache 2.0 License
+//    (found in the LICENSE.Apache file in the root directory).
+
+#ifndef ROCKSDB_LITE
+#ifdef GFLAGS
+#include "tools/io_tracer_parser_tool.h"
+
+#include <cinttypes>
+#include <cstdio>
+#include <iomanip>
+#include <memory>
+#include <sstream>
+
+#include "port/lang.h"
+#include "trace_replay/io_tracer.h"
+#include "util/gflags_compat.h"
+
+using GFLAGS_NAMESPACE::ParseCommandLineFlags;
+
+DEFINE_string(io_trace_file, "", "The IO trace file path.");
+DEFINE_string(
+    output_file, "",
+    "The output file path for storing human readable format of IO trace "
+    "records that are stored in IO trace file.");
```

Comment:
Would be better to describe the format in the command line

Reply:
In command line using -help?

Reply:
Yes, something like that.

Reply:
It does print the command line arguments when run 
 ```./io_tracer_parser --help ```

  Flags from tools/io_tracer_parser_tool.cc:   
-io_trace_file (The IO trace file path.) type: string default: ""